src/http-launch.c

```diff
@@ -303,13 +303,19 @@ on_message (GstBus * bus, GstMessage * message, gpointer user_data)
       break;
     }
     case GST_MESSAGE_WARNING:{
-      gchar *debug;
-      GError *err;
+      gchar *debug = NULL;
+      GError *err = NULL;
 
       gst_message_parse_error (message, &err, &debug);
-      g_print ("Warning %s\n", err->message);
-      g_error_free (err);
-      g_free (debug);
+      //g_print ("Warning %s\n", err->message);
```

Comment:
Oops... the problem is that we should use gst_message_parse_warning() here, not gst_message_parse_error(). Copy and paste mistake.

Want to update the pull request?